### DISCUSSION TITLE
New release because of fixed issues with gapless playback?

### DISCUSSION THREAD

**Discussion by PatrickJosh** [strawberrymusicplayer/strawberry#1213] [General]
Hey,

I'd like to ask if it is possible to create a new release because of the fixed issues with gapless playback. In the version I have installed via the package manager, gapless playback basically does not work with non-local files, which is a huge issue for me (and probably others). I led me to basically not using Strawberry any more and always listening to music via my mobile, but that's not as convenient. In the changelog, I have seen that this issue is fixed already but not yet released. I have compiled the main branch of Strawberry myself now and I don't have any issues with gapless playback any more. Hence, could you maybe release a new version so that others can benefit from that as well?

Thanks for consideration.
Best, 
Joshua

**Comment by jonaski** [comment]
Can't release a new version in a while yet as there are too many changes just recently merged, just starting testing.
The gap-less playback bug has been there for 5 years anyway.

**Comment by jonaski** [comment]
And we can't release a new version without including all the changes, as some things breaks in Qt 6.5 and some in Qt 6.6 (QSharedMemory, QNetworkDiskCache, etc).
